python/cudf/cudf/core/column/column.py

```diff
@@ -71,13 +71,13 @@ def __reduce__(self):
         return (
             build_column,
             (
-                self.base_data,
+                self.data,
```

Comment:
Let's explicitly specify args by name here to avoid breaking everything like I did :)

Reply:
I don't think you're allowed to for pickling but I could be mistaken.

Reply:
Is it possible to pass something like dictionary ?

Reply:
It's possible by defining a helper function that builds a column given a dictionary, and returning that instead of `build_column`, but don't know if we want to go through the trouble.